File: Ejer2.java
import java.util.Arrays;
import java.util.Scanner;

public class Ejer2 {

	public static void main(String[] args) {
		int x[] = new int[4];
		Scanner leer = new Scanner(System.in);
		String sc []=leer.nextLine().split(" ");
		for (int i = 0; i < sc.length; i++) {
			x[i]=Integer.parseInt(sc[i]);
		}
		Arrays.sort(x);
		System.out.println(x[2]);

	}

}

File: Ejer2.js
process.stdin.on('data',function(data){
	var sc=data.toString().split(" ");
	var x = [0,0,0,0];
	for(var i=0; i<4; i++)
	{
		x[i]=parseInt(sc[i]);
	}
	x.sort((a,b)=>a-b);
	console.log(x[2]);
	process.exit();
});
